File: UI/StoryBoard/src/app/dtos/user-dto/team-dto.ts
export class TeamDto {
    id : number = 0;
    userEmail : string = "";
    idProject : number = 0;
    userType : string = "";
    availabilityTime : number = 0;

    static createFromFormValues(formValues: any): TeamDto {
        const team = new TeamDto();
        team.userEmail = formValues.userEmail;
        team.idProject = formValues.idProject;
        team.userType = formValues.userType;
        team.availabilityTime = formValues.availabilityTime;
        return team;
    }
}

File: WebAPI2/StoryBoard/Dto/TeamDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Dto;
public class TeamDto
{
    public int Id { get; set; }
    public string UserEmail { get; set; }
    public string UserType { get; set; }
    public int AvailabilityTime { get; set; }
    public int IdProject { get; set; }

}
